sklearn/svm/_base.py

```diff
@@ -722,6 +726,7 @@ def __init__(
             C=C,
             nu=nu,
             epsilon=0.0,
+            quantile=0.0,
```

Comment:
Would it be possible to pass `quantile=None` in models for which the quantile parameter (and actually not pass it in the call to `super().__init__` when left to the default value).

Reply:
Now that you point it out, I'm not sure why I put this into BaseSVC, since that should be the base class for _classifiers_, and QSVR is not a classifier. I'll check if I can get rid of it in this place. Other than that, the module is just built in this somewhat odd way in which the base classes anticipate the arguments of all possible sub-classes. `epsilon=0.0` is also passed every time, even though not every SVM uses it.

Reply:
This is probably because the libsvm C++ wrapper functions (`libsvm.fit` / `libsvm_sparse.libsvm_sparse_train`) expect a fixed list of mandatory arguments and only the base class is calling the wrapper internal API.

We could change it to pass `getattr(self, "quantile", 0.0)` instead of `self.quantile` and only set the `self.quantile` attribute in the `__init__` of the concrete `QuantileSVR` subclass. This way this attribute would not be set on unrelated classes such as `SVC`.

We could fix other sub-class specific attributes (e.g. `self.nu`) in a similar way but I would rather like to keep this PR focused on quantile regression.

Reply:
Alright, I did what you suggested and used `getattr` with a default value when calling the libsvm function, and removed the quantile attribute from the base classes.